com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -780,6 +842,22 @@ public void OnAfterDeserialize()
                 k_AssetPreviousVersion = k_AssetVersion;
                 k_AssetVersion = 5;
             }
+
+            if (k_AssetVersion < 6)
+            {
+                int value = (int)m_ShadowCascades;
+                if (value == 2)
```

Comment:
I undestand the code because I know what happened, however it will be wise to add a comment in case for future maintanance in this upgrade code.

Reply:
Done